### PR TITLE
Added descriptions and syntax for $value.

### PR DESCRIPTION
**Pull request by angelgolfer-ms**

### CODE REVIEW

File: api-reference/beta/api/attachment-get.md [hunk 1]
```diff
@@ -19,6 +21,11 @@ An attachment can be one of the following types:
 All these types of attachment resources are derived from the [attachment](../resources/attachment.md)
 resource.
 
+### Get the raw contents of a file or item attachment
+You can append the path segment `/$value` to get the raw contents of a file or item attachment. For a file attachment, the content type is based on its original content type. For a message-type item attachment, the raw contents include the MIME representation of that message. 
+
+Attempting to get the `$value` of a reference attachment returns HTTP 405.
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/attachment-get.md]: @svpsiva , can you please confirm if HTTP 405 is the intended response code, as opposed to HTTP 501 as you mentioned before? I got HTTP 405 in Graph Explorer.

Comment (svpsiva) [reply api-reference/beta/api/attachment-get.md]: Hi,

405 is the intended response code.

Comment (svpsiva) [reply api-reference/beta/api/attachment-get.md]: 405 is the correct response code. Pls correct.

File: api-reference/beta/api/attachment-get.md [hunk 2]
```diff
@@ -19,6 +21,11 @@ An attachment can be one of the following types:
 All these types of attachment resources are derived from the [attachment](../resources/attachment.md)
 resource.
 
+### Get the raw contents of a file or item attachment
+You can append the path segment `/$value` to get the raw contents of a file or item attachment. For a file attachment, the content type is based on its original content type. For a message-type item attachment, the raw contents include the MIME representation of that message.
```
Comment (svpsiva) [inline api-reference/beta/api/attachment-get.md]: Item attachments can be of multiple types. They could be a Message, Contact, Event or Task. We can just change this to "For an item attachment, the raw contents include the MIME representation of that item. "

Comment (angelgolfer-ms) [reply api-reference/beta/api/attachment-get.md]: @svpsiva , I have further questions about the format of the raw content for item types other than message. For example, do contact attachments get returned in MIME format? I showed an example in an email to you on 4/8. Please take a look and re-confirm.

**Pull request merged**